### PR TITLE
Hunting multiple data sources

### PR DESCRIPTION
**Pull request by juliango2100**
Fixes #

## Proposed Changes

  - Fixed Tactics
  - Added Account entity
  - Added IP entity

### CODE REVIEW

File: Hunting Queries/MultipleDataSources/CobaltDNSBeacon.txt [hunk 1]
```diff
@@ -7,18 +7,21 @@
...
 // DataSource: Syslog, VMConnection , DnsEvents
 //
-// Tactics: #C2
+// Tactics: #CommandAndControl
 //
 (union isfuzzy=true
 (
 DnsEvents
 | where TimeGenerated >= ago(1d) 
+| where TimeGenerated <= ago(14d)
```
Comment (timbMSFT) [inline Hunting Queries/MultipleDataSources/CobaltDNSBeacon.txt]: logic seems off here -- >= ago(1d) means in the last day.
The modified query will return no results as <=ago(14d) (more than 2 weeks ago) has no overlap with the first restriction.

File: Hunting Queries/MultipleDataSources/CobaltDNSBeacon.txt [hunk 2]
```diff
@@ -7,18 +7,21 @@
...
 | where TimeGenerated >= ago(1d) 
+| where TimeGenerated <= ago(14d) 
 | where Name has "aaa.stage." or Name has "post.1"
 | extend Domain = Name, SourceIP = ClientIP , DestinationIp = IPAddresses
+| extend IPCustomEntity = IPAddresses
 ),
 (
 VMConnection
 | where TimeGenerated >= ago(1d)
+| where TimeGenerated <= ago(14d)
```
Comment (timbMSFT) [inline Hunting Queries/MultipleDataSources/CobaltDNSBeacon.txt]: logic seems off here -- >= ago(1d) means in the last day.
The modified query will return no results as <=ago(14d) (more than 2 weeks ago) has no overlap with the first restriction.

Comment (juliango2100) [reply Hunting Queries/MultipleDataSources/CobaltDNSBeacon.txt]: fixed

File: Hunting Queries/MultipleDataSources/CobaltDNSBeacon.txt (hunk 1 above)
Comment (juliango2100) [reply Hunting Queries/MultipleDataSources/CobaltDNSBeacon.txt]: fixed

**Comment by juliango2100** [comment]
Fixed time range issues from code review.  Merging to master

**Pull request merged**